app/controllers/patients_controller.rb

```diff
@@ -33,30 +31,22 @@ def update
     end
   end
 
-  # The following two methods are for mass data entry and
-  # should be turned off when not in use
   def data_entry
     @patient = Patient.new
-    @pregnancy = @patient.build_pregnancy
   end
 
-  def data_entry_create # temporary
+  def data_entry_create
     @patient = Patient.new patient_params
     @patient.created_by = current_user
-    @pregnancy = @patient.pregnancy || @patient.build_pregnancy
-    @pregnancy.created_by = current_user
-    (@patient.fulfillment || @patient.build_fulfillment).created_by = current_user
 
     if @patient.save
       flash[:notice] = "#{@patient.name} has been successfully saved! Add notes and external pledges, confirm the hard pledge and the soft pledge amounts are the same, and you're set."
-      current_user.add_patient @patient
```

Comment:
I don't understand this line's deletion. It doesn't seem related to pregnancy or fulfillment objects?